### DISCUSSION TITLE
Applying damage to tokens

### DISCUSSION THREAD

**Discussion by akosprime** [fvtt-fria-ligan/twilight2000-foundry-vtt#90] [Q&A]
I've figured out how to add equipment to an actor, and then how to cause an actor to utilize that equipment (so far just direct fire weapons) in combat to roll. I can see the ammo deducted from the token's sheet, but I never see damage applied to the target. Is that supposed to happen automatically? Or am I supposed to be manually applying it each time?

Also, related to this, are the results of penetration and criticals supposed to be automatically applied, or at least shown?

**Comment by Stefouch** [comment]
Hi there!
To apply damage to a character, you either need to do it manually in the character sheet, or select its token, then right-click the roll result message in the chat log and select "apply damage"

**Reply by tinwe** [reply]
> Hi there! To apply damage to a character, you either need to do it manually in the character sheet, or select its token, then right-click the roll result message in the chat log and select "apply damage"

No way of having it applied automatically if the target is already selected, like what I believe is happening in the Alien module?
